app/_locales/tr/messages.json

```diff
@@ -2089,6 +2079,9 @@
   "newNetworkAdded": {
     "message": "\"$1\" başarılı bir şekilde eklendi!"
   },
+  "newNftAddedMessage": {
+    "message": "Tahsil edilebilir tutar başarılı bir şekilde eklendi!"
```

Comment:
Should this be translated? Google Translate tells me "edilebilir" means "collectible", but I'm not sure what the correct translation here is.